app/controllers/rents_controller.rb

```diff
@@ -14,33 +12,27 @@ def main
   end
 
   def new
-    @TOA = Document.where(title: "Regler för Beerit").first
-    @rent = Rent.new
-    if (current_user)
-      @rent.prepare(current_user.profile)
-      @utskott = current_user.profile.car_councils
-    end
+    @TOA = Document.find_by(title: "Regler för Beerit")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Use snake_case for variable names.